.github/workflows/release.yml

```diff
@@ -42,24 +35,31 @@ jobs:
       runs-on: ubuntu-18.04
       needs: create-release-branch
       env:
-        GIT_AUTHOR_NAME: mparticle-bot
+        GITHUB_TOKEN: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
+        GIT_AUTHOR_NAME: mparticle-automation
         GIT_AUTHOR_EMAIL: developers@mparticle.com
-        GIT_COMMITTER_NAME: mparticle-bot
+        GIT_COMMITTER_NAME: mparticle-automation
         GIT_COMMITTER_EMAIL: developers@mparticle.com
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       steps:
         - name: Checkout public master branch
           uses: actions/checkout@v2
           with:
             fetch-depth: 0
-            repository: ${{ github.repository }}
-            token: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
             ref: master
+        
+        - name: Import GPG Key
+          uses: crazy-max/ghaction-import-gpg@v4
+          with:
+            gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+            passphrase: ${{ secrets.GPG_PASSPHRASE }}
```

Comment:
these secrets don't exist yet.  they exist in our mParticle 1P as a secure note (search for `gpg1` in 1P).  please add them as secrets

Reply:
Done!